hbase-server/src/main/java/org/apache/hadoop/hbase/util/TableDescriptorChecker.java

```diff
@@ -185,11 +175,6 @@ public static void sanityCheck(final Configuration c, final TableDescriptor td)
         warnOrThrowExceptionForFailure(logWarn, message, null);
```

Comment:
This is the place where we use logWarn in this method? Then let's move the `boolean logWarn = !shouldSanityCheck(conf);` line here?

Reply:
There are multiple checks based on logWarn variable before this line (for e.g. [at line no 92](https://github.com/apache/hbase/pull/4533/files#diff-e83f41eef4aa6cbdb757de3bcbd8c976a9068fea4849b4a52966ddad7a8018daR92), 106, 127, 134 above)

Reply:
Ah, OK, in unchanged code.